packages/fxa-settings/src/lib/hooks.tsx

```diff
@@ -2,7 +2,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { useEffect, useRef } from 'react';
+import { useEffect, useRef, createContext } from 'react';
+import { RelierData } from '../models/reliers/rellier';
```

Comment:
filename should be "../models/reliers/relier"?

Reply:
Or, `../models/relier` . Thanks for the call out regardless. 🙇‍♂️